### PR TITLE
[type:feat]add rocketmq logging e2e test

### PR DESCRIPTION
**Pull request by xcsnx**
Make sure that:

- [ ] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: .github/workflows/e2e-k8s.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ on:
   pull_request:
   push:
     branches:
-      - master
```
Comment (moremind) [inline .github/workflows/e2e-k8s.yml:22]: not change this

File: .github/workflows/e2e-k8s.yml [hunk 2]
```diff
@@ -109,7 +109,7 @@ jobs:
     needs:
       - changes
       - build-docker-images
-    if: (github.repository == 'apache/shenyu' && ${{ needs.changes.outputs.e2e == 'true' }})
+    if: (github.repository == 'xcsnx/shenyu' && ${{ needs.changes.outputs.e2e == 'true' }})
```
Comment (moremind) [inline .github/workflows/e2e-k8s.yml:112]: not change this

File: .github/workflows/e2e-k8s.yml [hunk 3]
```diff
@@ -199,7 +199,7 @@ jobs:
...
```
Comment (moremind) [inline .github/workflows/e2e-k8s.yml:202]: not change this

File: .github/workflows/e2e-k8s.yml [hunk 4]
```diff
@@ -281,11 +281,12 @@ jobs:
...
           sudo k3s ctr images import /tmp/shenyu/apache-shenyu-bootstrap.tar
 
-#      - name: Setup Debug Session
-#        uses: mxschmitt/action-tmate@v3
-#        timeout-minutes: 15
-#        with:
-#          detached: true
+
+      - name: Setup Debug Session
+        uses: mxschmitt/action-tmate@v3
```
Comment (moremind) [inline .github/workflows/e2e-k8s.yml:286]: comment this

**Review by moremind — CHANGES_REQUESTED** [review]

File: shenyu-e2e/shenyu-e2e-case/shenyu-e2e-case-http/k8s/rocket-mq.yml [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+      app: rocketmq-broker
+  template:
+    metadata:
+      labels:
+        app: rocketmq-broker
+    spec:
+      containers:
+        - name: broker
+          env:
+            - name: NAMESRV_ADDR     #通过环境变量指定namesrv 地址，注意该值为namesrv svc地址
```
Comment (moremind) [inline shenyu-e2e/shenyu-e2e-case/shenyu-e2e-case-http/k8s/rocket-mq.yml:37]: use English

**Review by moremind — CHANGES_REQUESTED** [review]

File: shenyu-e2e/shenyu-e2e-case/shenyu-e2e-case-http/k8s/rocket-mq.yml (hunk 5 above)
Comment (xcsnx) [reply shenyu-e2e/shenyu-e2e-case/shenyu-e2e-case-http/k8s/rocket-mq.yml]: ok

File: shenyu-e2e/shenyu-e2e-case/shenyu-e2e-case-http/k8s/script/e2e-http-sync.sh [hunk 6]
```diff
@@ -27,13 +27,19 @@ CUR_PATH=$(readlink -f "$(dirname "$0")")
 PRGDIR=$(dirname "$CUR_PATH")
 kubectl apply -f "${SHENYU_TESTCASE_DIR}"/k8s/sync/shenyu-cm.yml
 
+
```
Comment (moremind) [inline shenyu-e2e/shenyu-e2e-case/shenyu-e2e-case-http/k8s/script/e2e-http-sync.sh:30]: remove the line

File: shenyu-e2e/shenyu-e2e-case/shenyu-e2e-case-http/k8s/script/e2e-http-sync.sh [hunk 7]
```diff
@@ -27,13 +27,19 @@ CUR_PATH=$(readlink -f "$(dirname "$0")")
...
+
+
```
Comment (moremind) [inline shenyu-e2e/shenyu-e2e-case/shenyu-e2e-case-http/k8s/script/e2e-http-sync.sh:32]: remove the line

**Review by moremind — CHANGES_REQUESTED** [review]

**Review by moremind — APPROVED** [review]

**Pull request merged**